packages/gatsby-source-contentful/src/__tests__/normalize.js

```diff
@@ -55,7 +55,7 @@ describe(`Process contentful data`, () => {
     const createNodeId = jest.fn()
     createNodeId.mockReturnValue(`uuid-from-gatsby`)
     contentTypeItems.forEach((contentTypeItem, i) => {
-      normalize.createContentTypeNodes({
+      normalize.createNodesForContentType({
```

Comment:
I felt that this name better explained what it's doing — i.e., it's not _just_ creating the content type node itself (e.g., `Article`), but also the entries in the content type.

Reply:
Yeah, thanks for that - I was confused quite few times by the name of the function in the past ;)